### PR TITLE
Removing color from deployed app console logs

### LINKED ISSUE #434: Disable color in logs for deployed apps
### Describe the bug

We pass an env-var to dotnet applications when doing `tye run` to disable colorization via ANSI escape codes for the console logger.

We don't do this today for deployed applications.

### To Reproduce

Deploy something, and look at the log output.

### PR DESCRIPTION
**Pull request by QueenCitySmitty**
Fixes #434

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=577) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=577) before we can accept your contribution.<br/><br/>:x: QueenCitySmitty [sign now](https://cla.dotnetfoundation.org/dotnet/tye?pullRequest=577)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/tye?pullRequest=577) it.</sub>

File: src/Microsoft.Tye.Core/DockerfileGenerator.cs [hunk 1]
```diff
@@ -138,6 +141,9 @@ public static void ApplyContainerDefaults(ApplicationBuilder application, Docker
             }
 
             container.ImageTag ??= "latest";
+
+            // Disable color in the logs
+            project.EnvironmentVariables.Add(new EnvironmentVariableBuilder("DOTNET_LOGGING__CONSOLE__DISABLECOLORS") { Value = "true" });
```
Comment (davidfowl) [inline src/Microsoft.Tye.Core/DockerfileGenerator.cs:146]: This shouldn't be added for all docker files, just .NET projects.

Comment (QueenCitySmitty) [reply src/Microsoft.Tye.Core/DockerfileGenerator.cs:146]: Ah yeah that's pretty obvious now, will fix

**Comment by jkotalik** [comment]
Thanks!

**Pull request merged**